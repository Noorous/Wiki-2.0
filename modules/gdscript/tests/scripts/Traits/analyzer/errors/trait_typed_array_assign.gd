trait SomeTrait:
	pass

class SomeClass:
	uses SomeTrait

class OtherClass:
	pass

func test():
	var trait_classes: Array[SomeTrait] = [SomeClass.new(), OtherClass.new()]
