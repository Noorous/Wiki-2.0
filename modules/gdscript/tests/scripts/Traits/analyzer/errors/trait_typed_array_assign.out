GDTEST_ANALYZER_ERROR
Cannot have an element of type "OtherClass" in an array of type "Array[SomeTrait]".
